src/telemetry/types.ts

```diff
@@ -0,0 +1,50 @@
+export namespace Telemetry {
+  export interface Base {
```

Comment:
Since we're not using generics now, maybe `Telemetry.Base` should be `Telemetry.Provider`?

I had only used `Base` before to match the existing pattern w/ generics.

Reply:
Yeah, I think that makes more sense 👍🏻